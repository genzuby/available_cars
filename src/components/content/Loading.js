// @flow
import React from 'react';
import type { Node } from 'react';
import { Spinner, Container } from 'react-bootstrap';
import { mainStyle } from '../../style/componentStyle';

function Loading(): Node {
  return (
    <Container
      fluid
      style={mainStyle}
      className="d-flex flex-column justify-content-center align-items-center"
      data-testid="notFound"
    >
      <Spinner
        animation="border"
        role="status"
        data-testid="loading"
        style={{ width: '100px', height: '100px' }}
      >
        <span className="sr-only">Loading...</span>
      </Spinner>
    </Container>
  );
}

export default Loading;
