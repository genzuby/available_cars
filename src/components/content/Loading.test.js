import { render } from '@testing-library/react';
import Loading from './Loading';

it('render snapshot', () => {
  const { getByTestId } = render(<Loading />);
  expect(getByTestId('loading')).toMatchSnapshot();
});
