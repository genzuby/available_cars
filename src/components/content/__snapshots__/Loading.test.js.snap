// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render snapshot 1`] = `
<div
  class="spinner-border"
  data-testid="loading"
  role="status"
  style="width: 100px; height: 100px;"
>
  <span
    class="sr-only"
  >
    Loading...
  </span>
</div>
`;
